src/Microsoft.OData.Edm/Validation/EdmError.cs

```diff
@@ -25,12 +25,31 @@ public EdmError(EdmLocation errorLocation, EdmErrorCode errorCode, string errorM
             this.ErrorLocation = errorLocation;
             this.ErrorCode = errorCode;
             this.ErrorMessage = errorMessage;
+            this.ErrorLevel = EdmErrorLevel.Error;

```

Comment:
Could have this constructor call the new constructor with the error level auto-set to in this case "Error", reduces duplication and ease of maintenance.

Reply:
This has been done.